ci-operator/step-registry/upi/install/gcp/upi-install-gcp-commands.sh

```diff
@@ -601,6 +601,20 @@ else # for workflow before internal load balancers
   gcloud compute target-pools remove-instances "${INFRA_ID}-ign-target-pool" "--instances-zone=${ZONE_0}" "--instances=${INFRA_ID}-bootstrap"
   gcloud compute target-pools remove-instances "${INFRA_ID}-api-target-pool" "--instances-zone=${ZONE_0}" "--instances=${INFRA_ID}-bootstrap"
 fi
+# remove from the backend service of random id
+backend_service_random_id=$(gcloud compute backend-services list --format=json --filter="backends[].group~${INFRA_ID}" | jq -r .[].name | grep -v "${INFRA_ID}")
```

Comment:
When `backend_service_random_id` is empty, the command will return non-zero, the line will cause the whole process exit as failure. Refer to https://prow.ci.openshift.org/view/gs/test-platform-results/pr-logs/pull/openshift_release/55605/rehearse-55605-periodic-ci-openshift-release-master-nightly-4.14-e2e-gcp-ovn-upi/1824293917588197376